### PR TITLE
Add getfocusableaffs()

### PR DESCRIPTION
**Pull request by bunster**
Addresses issue #155

### CODE REVIEW

**Comment by keneanung** [comment]
Additions look good. But the committer info doesn't match your account, so the commits don't refer back to you. It might be enough to add the e-mail you used as author e-mail to your GitHub as a secondary mail address

**Comment by bunster** [comment]
My apologies. Forgot to reset my primary email. I've done that now.

File: svo (install the zip, not me).xml [hunk 1]
```diff
@@ -54122,6 +54122,11 @@ function svo.gettreeableaffs()
   return table.n_intersection(svo.keystolist(svo.affl), svo.treecurables)
 end
 
+-- returns a list of affs that fous can get
```
Comment (bunster) [inline svo (install the zip, not me).xml]: Accidentally misspelled focus here. I can fix this if it is preferable...

Comment (keneanung) [inline svo (install the zip, not me).xml]: That'd be great. And while we are at documentation and comments... Could you add the new function to svof/doc/index.rst as well?

**Comment by vadi2** [comment]
+1

**Pull request merged**